### PR TITLE
Selenium homework

### PR DESCRIPTION
**Pull request by GlebEmelianov**
Tests for https://litecart.stqa.ru/en/
Tests for shadow DOM, frames, tabs, web storage

### CODE REVIEW

**Comment by GlebEmelianov** [comment]
Added Allure and Report Portal reports

File: src/test/java/helpers/Browsers.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+public enum Browsers {
+        chrome("chrome"),
+        firefox("firefox"),
+        opera("opera"),
+        safari("safari"),
+        ie("internet explorer");
+        String browserName;
+        Browsers(String browserName) {
+            this.browserName = browserName; }
```
Comment (Fulcrum81) [inline src/test/java/helpers/Browsers.java]: Please move the closing brace to new line

File: src/test/java/otherTests/linkTest.java [hunk 2]
```diff
@@ -14,5 +15,6 @@ public void linkTest() {
         driver.get("https://the-internet.herokuapp.com/");
         List<WebElement> allLinks = driver.findElements(By.tagName("a"));
         allLinks.get(10).click();
+        driver.quit();
```
Comment (Fulcrum81) [inline src/test/java/otherTests/linkTest.java]: Here and in all other tests WebDriver container is not used. Will not work that way

File: src/test/java/otherTests/theInternetTests.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class theInternetTests {
+    @Test
+    public void frameTest() {
+        WebDriver driver = new ChromeDriver();
```
Comment (Fulcrum81) [inline src/test/java/otherTests/theInternetTests.java]: Why are we instantiating driver in tests? firstly there's the WebDriverContainer for that, secondly there's the setup and teardown to contain opening and closing driver

**Comment by GlebEmelianov** [comment]
Corrected mistakes according to comments above

File: src/test/java/otherTests/theInternetTests.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
+package otherTests;
+
+import helpers.WebDriverContainer;
+import org.openqa.selenium.*;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class theInternetTests extends WebDriverContainer {
```
Comment (Fulcrum81) [inline src/test/java/otherTests/theInternetTests.java]: No need for this inheritance

File: src/test/java/pageobject/objectmethod/homePage.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
+package pageobject.objectmethod;
+
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.By;
+
+
+
+public class homePage  {
```
Comment (Fulcrum81) [inline src/test/java/pageobject/objectmethod/homePage.java]: Here and in all other classes: Class names should start with capital letters

**Comment by GlebEmelianov** [comment]
Corrected the code according to two comments above

**Pull request merged**